config/routes.yaml

```diff
@@ -87,6 +87,10 @@ app_user_download_personal_data:
   path: /{_locale}/user/personal-data-download
   controller: App\Controller\User::downloadPersonalData
 
+app_place:
+  path: /{_locale}/place
```

Comment:
This path is not allowing access to `/{_locale}/place/new`. Can we change it to something like `/{_locale}/place/list`?

Reply:
Changed to /{_locale}/places.